IdentifiabilityODE.mpl

```diff
@@ -277,10 +278,36 @@ IdentifiabilityODE := proc(system_ODEs, params_to_assess, {p := 0.99, count_solu
     z_aux, w_aux,
     op(sort(mu))
   ]:
+
+  ###########
+  non_id := map(x -> ParamToOuter(x, all_vars), [op({op(theta)} minus {op(theta_l)})]):
+  if use_weights then
+  	if infolevel > 0 then
+    		PrintHeader("Applying Weighted Ordering", output_targets[log]):
+    		LogText(sprintf("\t=> Applying Weighted Ordering"), ProgressBar):
+  	end if:
+    input_table := table(
+      [
+        "sigma"=system_ODEs,
+        "poly_system" = [op(Et_hat), z_aux * Q_hat - 1], "poly_vars" = vars, "non_id" = non_id, 
+        "s"=s, "m"=m, "x_vars"=x_vars, "y_vars"=y_vars,
+        "mu"=mu, "x_eqs"=x_eqs, "y_eqs"=y_eqs, "all_vars"=all_vars
+      ]
+    );
+    weight_subs, poly_system := SubsByDepth(input_table):
+    Et_hat := poly_system;
+    weights_table := table(weight_subs);
+  else
+    weights_table := table([seq(_var_=_var_, _var_ in vars)]);
+  	Et_hat := [op(Et_hat), z_aux * Q_hat - 1]:

```

Comment:
Something with indentation

Reply:
I am not sure what it is, I see it normally in this part of the code.

Reply:
Lines 301 and 302 have different indentation, don't they?

Reply:
You're right, there was a tab somewhere in there, I think. It was not showing the difference on my editor, so I never noticed. Thanks!